### ISSUE TITLE
[PlantUML] Local Includes Do Not Work

### ISSUE THREAD

**Issue by Jeeppler** [de-jcup/eclipse-asciidoctor-editor#318]
I am currently creating [C4 diagrams](https://github.com/RicardoNiepel/C4-PlantUML). I need to include the supporting C4-PlantUML files to get the C4 elements. The remote include works well. However, the local include does not work. 

Example, `context.puml`:
~~~
@startuml context
!include C4_Context.puml

Person(administrator, "Administrator")
System(jenkins, "Jenkins")
Rel(administrator, jenkins, "Administrates Jenkins", "SSH")

@enduml
~~~

The reason is, that the Asciidoctor Editor Eclipse Plugin does not copy the included files from `context.puml` into the temporary asciidoctor folder. The workaround is, to copy the included files manually into the temporary asciidoctor folder. 

~~~
/tmp/asciidoctor-editor-temp/my-project-1149858479$ ls -1
-2127571943_container.html
-2127571943_container.puml
-2127571943_internal_container.html
-2127571943_preview_container.html
C4_Component.puml
C4_Container.puml
C4_Context.puml
C4.puml
images
~~~

A solution would be to look in the PlantUML files for the `!include` directive and follow the include. This has to be done recursively. It has to be done recursively, because the include chain in the example `context.puml` looks like this:

`context.puml` ->  `C4_Context.puml` -> `C4.puml`

All the included files have to be copied over. In case an included file cannot be found, the Asciidoctor Editior Plugin should throw an exception (show a dialog) an inform the user about the missing file.

**Comment by de-jcup** [comment]
Just released 2.3.0 to eclipse marketplace, your example works now:

![image](https://user-images.githubusercontent.com/11644753/94646743-3421c600-02ef-11eb-8bbc-5800684eb95a.png)

**Comment by de-jcup** [comment]
Just for information about C4 model see https://c4model.com/
